### PR TITLE
Additional schema fields; ref retsly/import#360

### PR DESCRIPTION
**Pull request by lvivier**
ref retsly/import#360

### CODE REVIEW

**Comment by switters** [comment]
👍 build went off the deep end?

**Comment by lvivier** [comment]
@AndrewMartens @CMClay review please?

**Comment by CMClay** [comment]
👍

File: lib/listing-desc.js [hunk 1]
```diff
@@ -80,6 +89,8 @@ module.exports = {
   garageSpaces: 'The number of spaces in the garage if applicable.',
   carportSpaces: 'The number of spaces in the carport if applicable.',
   openParking: 'A boolean indicating if there is open space parking.',
+  attachedGarage: 'A flag indicating that the garage attached to the dwelling.',
```
Comment (AndrewMartens) [inline lib/listing-desc.js]: Grammar - "... that the garage *is* attached to the dwelling", perhaps ?

**Comment by AndrewMartens** [comment]
One suggested grammar change, otherwise 👍 good to go.

File: lib/listing-desc.js (hunk 1 above)
Comment (lvivier) [inline lib/listing-desc.js]: Sure! These are just transcribed from the Data Dictionary. I'll correct it.

**Pull request merged**